src/NzbDrone.Core/Notifications/Mailgun/MailgunSettings.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Notifications.Mailgun
+{
+    public class MailGunSettingsValidator : AbstractValidator<MailgunSettings>
+    {
+        public MailGunSettingsValidator()
+        {
+            RuleFor(c => c.ApiKey).NotEmpty();
+            RuleFor(c => c.From).NotEmpty();
```

Comment:
I don’t know anything about mailgun, but is this really required for a call? Or is it just an override like email?

Reply:
Seems this is good per docs

Reply:
Excellent! I wasn't sure but it was part of the pull so I assumed it was needed